### PR TITLE
3027 - Investigate Queue-Consumers error in Prod

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Change to remove `RoleBinding` from `make delete-redis` command as we don't need that in our environments and it causes an error:
image

After the change:
image

- Changed spacing of `storageClassName` to make it be under `spec`. That issue cause an error when testing on TEST env:
image

### CODE REVIEW

File: devops/Makefile [hunk 1]
```diff
@@ -179,7 +179,7 @@ print-status:
 	@echo " +---------------------------------------------------------+ "
 
 init-project:
-	@echo "Run initial set-up for the project, including network security\n"
```
Comment (dheepak-aot) [inline devops/Makefile:182]: What is the reason behind formatting changes here?

File: devops/Makefile [hunk 2]
```diff
@@ -615,4 +615,4 @@ db-backup-deploy-patroni-delete:
 # Remove redis and resources including secrets from openshift namespace.
 delete-redis:
 	@echo "+\n++ Removing redis and resources on $(NAMESPACE)\n+"
-	@oc delete -n $(NAMESPACE) all,rc,svc,dc,route,pvc,secret,configmap,sa,RoleBinding -l app=redis
```
Comment (dheepak-aot) [inline devops/Makefile:618]: 👍

File: devops/Makefile (hunk 1 above)
Comment (andrepestana-aot) [reply devops/Makefile:182]: Comparing the files it seems that there are spaces before the carriage return at some end of lines: 
image
image
image
image

Comment (andrewsignori-aot) [reply devops/Makefile:182]: Checking the file in my local and saving the PR-changed file did not cause additional changes.
It seems that the PR file has the right formatting.

File: devops/openshift/database/redis-ha-deploy.yml [hunk 3]
```diff
@@ -147,7 +147,7 @@ objects:
             resources:
               requests:
                 storage: ${PVC_SIZE}
-          storageClassName: ${STORAGE_CLASS}
+            storageClassName: ${STORAGE_CLASS}
```
Comment (andrewsignori-aot) [inline devops/openshift/database/redis-ha-deploy.yml:150]: Just to support the change, `storageClassName` is defined under the specs as per the below doc.
https://docs.openshift.com/container-platform/4.13/rest_api/storage_apis/persistentvolumeclaim-v1.html#spec
Thanks for making the change.

**Review by andrewsignori-aot — APPROVED** [review]
Nice work, thanks for the investigation, looks good 👍

File: devops/openshift/database/redis-ha-deploy.yml (hunk 3 above)
Comment (andrepestana-aot) [reply devops/openshift/database/redis-ha-deploy.yml:150]: I redid a test on DEV and it worked both ways. 😄 That's good that we tested on TEST.

Comment (dheepak-aot) [inline devops/openshift/database/redis-ha-deploy.yml:150]: Thanks for taking care of this. 👍

**Review by dheepak-aot — APPROVED** [review]
Great analysis and work 👍

File: devops/openshift/database/redis-ha-deploy.yml (hunk 3 above)
Comment (andrepestana-aot) [reply devops/openshift/database/redis-ha-deploy.yml:150]: I went back to the history now and checked that it did happen on DEV but I didn't pay much attention it was just a warning.
image

**Pull request merged**